### PR TITLE
Report a problem frontend functionality

### PR DESCRIPTION
**Pull request by khusbupatel**
#99 

The image is successfully stored in Azure, however on refresh the uploaded image disappears from the frontend and shows "No File Chosen" even though a file is in fact stored. Is there a way to obtain the files stored and display them on the page again?

### CODE REVIEW

File: server/utils/ImageUpload.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { BlobServiceClient, ContainerClient } from "@azure/storage-blob";
+
+const sasToken =
+  process.env.storagesastoken ||
+  "sv=2019-12-12&ss=b&srt=sco&sp=rwdlacx&se=2021-02-09T08:03:52Z&st=2021-02-09T00:03:52Z&spr=https&sig=huSsjMTIgztooX2xhIfFsXDxAeiLINye1%2FCzyu3%2Fnh4%3D"; // Fill string with your SAS token
```
Comment (matthewbrown77) [inline server/utils/ImageUpload.ts:5]: This SAS token will need to be put into bitwarden or an equivalent. I can regen a new one that doesn't expire so just let me know when you need that done @Sukhmai

File: src/components/ImageUpload/index.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+interface Props {
+  setImageUrl: React.Dispatch<React.SetStateAction<string>>;
+}
+
+const ImageUpload: React.FC<Props> = ({ setImageUrl }) => {
+  const [fileSelected, setFileSelected] = useState(null);
+  const [uploading, setUploading] = useState(false);
+
+  const onFileChange = (event: any) => {
+    setFileSelected(event.target.files[0]);
```
Comment (matthewbrown77) [inline src/components/ImageUpload/index.tsx:15]: This is the part I reached out about. We want to pass the url of the previously uploaded image when a user has saved a form for later, but this `setFileSelected` function is what controls the preview and it needs an actual file.

File: server/utils/ImageUpload.ts (hunk 1 above)
Comment (matthewbrown77) [reply server/utils/ImageUpload.ts:5]: Actually we probably need to switch all the credentials out for the bog account (if there is one for azure).

**Comment by Sukhmai** [comment]
Hey Guys! I'm currently working on getting the bog azure account + adding the SAS token to the bitwarden so I can test the upload feature and see if I can get the image to display from localstorage. 

In the mean time you guys could work on getting the frontend more in line with the design.  
Currently it looks like this on my screen:
2021-02-09-181907_2560x1440_scrot
The old design had a large grey box for it, but I would reach out to one of the designers and ask them to make a mockup for it, as I don't love the old design.

Otherwise y'all can continue to try debug or just chill until sprint 2.

**Comment by matthewbrown77** [comment]
I think it would be best to wait until the design gets into Figma. It didn't seem like a preview was necessary with the new design, so we can probably get away with just displaying the names of uploaded files in the popup.

**Review by Sukhmai — APPROVED** [review]
Great job! The design matches the figma perfectly, and the image is correctly uploaded to the bog blob storage container when I manually set the SAS token.

**Pull request merged**